package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "testcafe-functional-harness",
```

Comment:
So, after a rethink, I suppose we should call it `testcafe-functional-harness`. Because the running of the functional tests is specific for `testcafe` in our case.
So, we should replace `functional-harness` to `testcafe-functional-harness`.
We should have a description something like:
```
Allows to run TestCafe functional tests in local browsers or on SauceLabs.
```